jyulb/fileio/common/jyu_lattice_proxy.py

```diff
@@ -0,0 +1,123 @@
+from simphony.cuds.abstractlattice import ABCLattice
+from simphony.cuds.lattice import LatticeNode
+import numpy as np
+
+
+class JYULatticeProxy(ABCLattice):
+
+    """
+    A proxy lattice for accessing state data of a JYU-LB modeling engine.
+
+    Updates and queries of node data are relayed to external data storages.
+    Acknowledges only those CUBA keywords which are prescribed at the
+    initialization.
+
+    Parameters
+    ----------
+    name : str
+    type : str
+        Bravais lattice type (should agree with the base_vect below).
+    base_vect : D x float
+        defines a Bravais lattice (an alternative for primitive vectors).
+    size : tuple of D x size
+        number of lattice nodes (in the direction of each axis).
+    origin : D x float
+    data : dictionary
```

Comment:
Good idea, I will change it.

It is true that the current implementation of ABCLattice does not involve data attribute, and the standalone Lattice implementation has only private data (self._dcs) for the node data. I think that the data in the D1.3 class diagram refers to a private attribute for storing the node data ... Anyway, we could add data to ABCLattice interface: is there a clear motivation for such an attribute?